libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.h

```diff
@@ -109,19 +109,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, getter = isUserDeleted) BOOL userDeleted;
```

Comment:
All guest user functionality should go too (`initWithGuestUser`, `self.guestUser` guest user constants, etc.).